events-api/src/modules/events/events.controller.ts

```diff
@@ -23,3 +29,64 @@ EventsController.get(
     });
   }
 );
+
+EventsController.post(
+  "/",
+  validateRequestBody(eventsCreateDtoSchema),
+  async (req, res) => {
+    await EventsService.createEvent(req.body);
+
+    return res.status(201).json({
+      message: "Event created successfully",
+    });
+  }
+);
+
+EventsController.patch(
+  "/:id",
+  validateRouteParams(eventsRouteParamsDtoSchema),
+  validateRequestBody(eventsUpdateDtoSchema),
+  async (req, res) => {
+    const event = await EventsService.updateEvent(
+      req.body,
+      req.params["id"] as string
+    );
+    if (event === null) {
+      return res.status(404).json({ message: "Event not found" });
+    }
+
+    return res.status(201).json({
+      message: "Event updated successfully",
```

Comment:
This endpoint should also return a updated event data: https://github.com/NegatiV-64/expressjs-masterclass-2024/blob/main/homework/README.md?plain=1#L63

And also 201 is a wrong status code for the response. 201 is typically used with the POST API: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201

Reply:
![image](https://github.com/user-attachments/assets/318196d0-941e-4ee7-b738-dc7b15b88ffd)

Reply:
But what stops you from:

- Inserting a record
- Selecting a created record

Reply:
I thought that multiple request  to db is not good

Reply:
![image](https://github.com/user-attachments/assets/e45b08dc-8f37-47ca-87b9-06ef46a1c5df)
done

Reply:
Fix casing of the response. Right now it is inconsistent with other responses